packages/core/scripts/local/SnapshotCurrentRound.js

```diff
@@ -0,0 +1,20 @@
+const Voting = artifacts.require("Voting");
```

Comment:
Outside scope of this PR but this script was very helpful for me since latest Voting requires a snapshot from an EOA to reveal votes

Reply:
+1